gluecodium/src/main/resources/templates/dart/DartLibraryContext.mustache

```diff
@@ -71,6 +71,12 @@ enum IsolateOrigin {
   spawned
 }
 
+enum WaitCallbackResult {
+  stopped,
+  has_incoming,
```

Comment:
Dart enumerator names should be lowerCamelCase, not lower_snake_case: https://dart.dev/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase